.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+# 
+
+# Configuration file for Circle CI 
+# CI will report failure if any executed command returns and error status
+# Operations performed are as follows
+# Build source code
+# Run unit tests for C++ and Java
+# Run static code analyzer for SourceCloud
+# Upload test results
+# Every run command should start with source ${INIT_ENV} to ensure all default dependancies are available
+
+jobs:
+  build:
+    # Pull docker image from docker hub
+    # XTERM used for better catkin_make output
+    docker:
+      - image: usdotfhwastol/autoware.ai:3.1.0-Test
```

Comment:
update version tag

Reply:
@msmcconnell do we need to do this for all circle ci config yml files?

Reply:
Any circle config that says autoware.ai should be checked to make sure it points to 3.1.0 not 3.1.0-Test

Reply:
Yup, we got it. Thanks for catching that!